### DISCUSSION TITLE
Possible to use new Phasar 3.50 with "Bridge.NET/Retyped.Phasor"?

### DISCUSSION THREAD

**Discussion by CodingMadness** [photonstorm/phaser#5913] [Feature Requests]
**We're constantly trying to make Phaser 3 better**

Title.        
(Like how its over here but with the restriction as it seems that its only usable with Phasar 2 :/  
https://github.com/Retyped/Demos/tree/master/PhaserDemo)

**[Optional] Do you want to help provide this feature?**

If you would like to get involved in helping build the feature, please let us know. We can then discuss implementation with you and the best way to approach it. Again, screenshots or mockups are really useful.

**Comment by CodingMadness** [comment]
@gammafp I dont wanna hurry any1, but would you mind setting this goal up as high-priority since I have some ppl wanting to build up a 2.5D Game with it, and sell it on steam and other platforms, so we need this massive-new- Phasar 3.5 vers of it :D 

Would be really nice to have this possible for the next month or so :)

**Comment by photonstorm** [comment]
@Shpendicus I'm glad you're excited about the possibility of using Phaser 3.50 for your project. However, please understand that we had nothing to do with the Bridge.NET / Retyped project. To be honest, I never even knew it existed until you opened this issue! It wasn't created by the Phaser team, so your best bet is to reach out to the developer who did create it to find out what's involved in porting it to 3.50. Alternatively, you may wish to stick a Bounty on this feature request to see if anyone else can pick it up, as it's not something we can do internally I'm afraid. Good luck with your game.

**Comment by CodingMadness** [comment]
@photonstorm yea I did , i fear its a sort of dead project, which I really hope is not, since they didndt touch it since Phasar 2, which doesnt sound really promising ..
